src/aosm/azext_aosm/configuration_models/onboarding_nfd_base_input_config.py

```diff
@@ -25,6 +25,16 @@ class OnboardingNFDBaseInputConfig(OnboardingBaseInputConfig):
             "comment": "Version of the network function definition in 1.1.1 format (three integers separated by dots)."
         },
     )
+    expose_all_parameters: bool = field(

```

Comment:
not sure about this, in other cases for bools they are still strings. I cnt remember why we did this, maybe we should change the other instead of this